--- kmymoney-4.8.0/kmymoney/mymoney/storage/mymoneyseqaccessmgrtest.cpp.orig	2016-06-14 21:16:15.585035116 +0200
+++ kmymoney-4.8.0/kmymoney/mymoney/storage/mymoneyseqaccessmgrtest.cpp	2017-02-10 22:14:40.088737421 +0100
@@ -41,6 +41,11 @@
   delete m;
 }
 
+inline size_t sz(unsigned val)
+{
+  return static_cast<size_t>(val);
+}
+
 void MyMoneySeqAccessMgrTest::testEmptyConstructor()
 {
   MyMoneyPayee user = m->user();
@@ -58,13 +63,13 @@
   QCOMPARE(m->m_nextPayeeID, 0ul);
   QCOMPARE(m->m_nextScheduleID, 0ul);
   QCOMPARE(m->m_nextReportID, 0ul);
-  QCOMPARE(m->m_institutionList.count(), 0ul);
-  QCOMPARE(m->m_accountList.count(), 5ul);
-  QCOMPARE(m->m_transactionList.count(), 0ul);
-  QCOMPARE(m->m_transactionKeys.count(), 0ul);
-  QCOMPARE(m->m_payeeList.count(), 0ul);
-  QCOMPARE(m->m_tagList.count(), 0ul);
-  QCOMPARE(m->m_scheduleList.count(), 0ul);
+  QCOMPARE(m->m_institutionList.count(), sz(0u));
+  QCOMPARE(m->m_accountList.count(), sz(5u));
+  QCOMPARE(m->m_transactionList.count(), sz(0u));
+  QCOMPARE(m->m_transactionKeys.count(), sz(0u));
+  QCOMPARE(m->m_payeeList.count(), sz(0u));
+  QCOMPARE(m->m_tagList.count(), sz(0u));
+  QCOMPARE(m->m_scheduleList.count(), sz(0u));
 
   QCOMPARE(m->m_dirty, false);
   QCOMPARE(m->m_creationDate, QDate::currentDate());
