diff -urN kmymoney2-0.8.orig/kmymoney2/dialogs/kadvancedbanksettingsdlgdecl.ui kmymoney2-0.8/kmymoney2/dialogs/kadvancedbanksettingsdlgdecl.ui
--- kmymoney2-0.8.orig/kmymoney2/dialogs/kadvancedbanksettingsdlgdecl.ui	2005-11-02 00:50:18.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/dialogs/kadvancedbanksettingsdlgdecl.ui	2005-11-02 00:50:44.000000000 +0000
@@ -352,9 +352,4 @@
     </connection>
 </connections>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kurlrequester.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kmymoney2-0.8.orig/kmymoney2/dialogs/kbackupdlgdecl.ui kmymoney2-0.8/kmymoney2/dialogs/kbackupdlgdecl.ui
--- kmymoney2-0.8.orig/kmymoney2/dialogs/kbackupdlgdecl.ui	2005-11-02 00:50:18.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/dialogs/kbackupdlgdecl.ui	2005-11-02 00:50:44.000000000 +0000
@@ -206,9 +206,4 @@
 <customwidgets>
 </customwidgets>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kmymoney2-0.8.orig/kmymoney2/dialogs/kcurrencycalculatordecl.ui kmymoney2-0.8/kmymoney2/dialogs/kcurrencycalculatordecl.ui
--- kmymoney2-0.8.orig/kmymoney2/dialogs/kcurrencycalculatordecl.ui	2005-11-02 00:50:18.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/dialogs/kcurrencycalculatordecl.ui	2005-11-02 00:50:44.000000000 +0000
@@ -378,8 +378,4 @@
     <tabstop>m_cancelButton</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-</includehints>
 </UI>
diff -urN kmymoney2-0.8.orig/kmymoney2/dialogs/kcurrencyeditdlgdecl.ui kmymoney2-0.8/kmymoney2/dialogs/kcurrencyeditdlgdecl.ui
--- kmymoney2-0.8.orig/kmymoney2/dialogs/kcurrencyeditdlgdecl.ui	2005-11-02 00:50:18.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/dialogs/kcurrencyeditdlgdecl.ui	2005-11-02 00:50:44.000000000 +0000
@@ -301,7 +301,4 @@
     </image>
 </images>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kmymoneylineedit.h</includehint>
-</includehints>
 </UI>
diff -urN kmymoney2-0.8.orig/kmymoney2/dialogs/keditequityentrydecl.ui kmymoney2-0.8/kmymoney2/dialogs/keditequityentrydecl.ui
--- kmymoney2-0.8.orig/kmymoney2/dialogs/keditequityentrydecl.ui	2005-11-02 00:50:18.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/dialogs/keditequityentrydecl.ui	2005-11-02 00:50:44.000000000 +0000
@@ -303,8 +303,4 @@
     </image>
 </images>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneypriceview.h</includehint>
-</includehints>
 </UI>
diff -urN kmymoney2-0.8.orig/kmymoney2/dialogs/keditschedtransdlgdecl.ui kmymoney2-0.8/kmymoney2/dialogs/keditschedtransdlgdecl.ui
--- kmymoney2-0.8.orig/kmymoney2/dialogs/keditschedtransdlgdecl.ui	2005-11-02 00:50:18.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/dialogs/keditschedtransdlgdecl.ui	2005-11-02 00:50:44.000000000 +0000
@@ -855,19 +855,4 @@
     <tabstop>m_kdateinputFinal</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kmymoneyaccountcombo.h</includehint>
-    <includehint>kmymoneycombo.h</includehint>
-    <includehint>kmymoneycombo.h</includehint>
-    <includehint>kmymoneyaccountcombo.h</includehint>
-    <includehint>kmymoneypayee.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneycategory.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kmymoney2-0.8.orig/kmymoney2/dialogs/kendingbalancedlgdecl.ui kmymoney2-0.8/kmymoney2/dialogs/kendingbalancedlgdecl.ui
--- kmymoney2-0.8.orig/kmymoney2/dialogs/kendingbalancedlgdecl.ui	2005-11-02 00:50:18.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/dialogs/kendingbalancedlgdecl.ui	2005-11-02 00:50:44.000000000 +0000
@@ -1060,19 +1060,4 @@
     </image>
 </images>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneyaccountselector.h</includehint>
-    <includehint>kmymoneyaccountselector.h</includehint>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneycategory.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneycategory.h</includehint>
-</includehints>
 </UI>
diff -urN kmymoney2-0.8.orig/kmymoney2/dialogs/kenterscheduledialogdecl.ui kmymoney2-0.8/kmymoney2/dialogs/kenterscheduledialogdecl.ui
--- kmymoney2-0.8.orig/kmymoney2/dialogs/kenterscheduledialogdecl.ui	2005-11-02 00:50:18.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/dialogs/kenterscheduledialogdecl.ui	2005-11-02 00:50:44.000000000 +0000
@@ -655,12 +655,4 @@
     </connection>
 </connections>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kmymoneypayee.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneycategory.h</includehint>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kmymoneyaccountcombo.h</includehint>
-    <includehint>kmymoneyaccountcombo.h</includehint>
-</includehints>
 </UI>
diff -urN kmymoney2-0.8.orig/kmymoney2/dialogs/kexportdlgdecl.ui kmymoney2-0.8/kmymoney2/dialogs/kexportdlgdecl.ui
--- kmymoney2-0.8.orig/kmymoney2/dialogs/kexportdlgdecl.ui	2005-11-02 00:50:18.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/dialogs/kexportdlgdecl.ui	2005-11-02 00:50:44.000000000 +0000
@@ -504,14 +504,4 @@
     </image>
 </images>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kmymoneyaccountcombo.h</includehint>
-    <includehint>kcombobox.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kmymoney2-0.8.orig/kmymoney2/dialogs/kfindtransactiondlgdecl.ui kmymoney2-0.8/kmymoney2/dialogs/kfindtransactiondlgdecl.ui
--- kmymoney2-0.8.orig/kmymoney2/dialogs/kfindtransactiondlgdecl.ui	2005-11-02 00:50:18.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/dialogs/kfindtransactiondlgdecl.ui	2005-11-02 00:50:44.000000000 +0000
@@ -1302,17 +1302,4 @@
     </image>
 </images>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kmymoneyaccountselector.h</includehint>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneyaccountselector.h</includehint>
-    <includehint>kmymoneylineedit.h</includehint>
-    <includehint>kmymoneylineedit.h</includehint>
-    <includehint>kmymoneylineedit.h</includehint>
-    <includehint>kmymoneyregistersearch.h</includehint>
-</includehints>
 </UI>
diff -urN kmymoney2-0.8.orig/kmymoney2/dialogs/kimportdlgdecl.ui kmymoney2-0.8/kmymoney2/dialogs/kimportdlgdecl.ui
--- kmymoney2-0.8.orig/kmymoney2/dialogs/kimportdlgdecl.ui	2005-11-02 00:50:18.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/dialogs/kimportdlgdecl.ui	2005-11-02 00:50:44.000000000 +0000
@@ -386,13 +386,4 @@
 <customwidgets>
 </customwidgets>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kcombobox.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kcombobox.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kmymoney2-0.8.orig/kmymoney2/dialogs/kmymoneypricedlgdecl.ui kmymoney2-0.8/kmymoney2/dialogs/kmymoneypricedlgdecl.ui
--- kmymoney2-0.8.orig/kmymoney2/dialogs/kmymoneypricedlgdecl.ui	2005-11-02 00:50:18.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/dialogs/kmymoneypricedlgdecl.ui	2005-11-02 00:50:44.000000000 +0000
@@ -217,7 +217,4 @@
     <tabstop>m_closeButton</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kmymoneypriceview.h</includehint>
-</includehints>
 </UI>
diff -urN kmymoney2-0.8.orig/kmymoney2/dialogs/knewaccountdlgdecl.ui kmymoney2-0.8/kmymoney2/dialogs/knewaccountdlgdecl.ui
--- kmymoney2-0.8.orig/kmymoney2/dialogs/knewaccountdlgdecl.ui	2005-11-02 00:50:18.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/dialogs/knewaccountdlgdecl.ui	2005-11-02 00:50:44.000000000 +0000
@@ -763,11 +763,4 @@
     <tabstop>cancelButton</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kmymoneyequity.h</includehint>
-    <includehint>kmymoneycurrencyselector.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneyaccountselector.h</includehint>
-</includehints>
 </UI>
diff -urN kmymoney2-0.8.orig/kmymoney2/dialogs/knewaccountwizarddecl.ui kmymoney2-0.8/kmymoney2/dialogs/knewaccountwizarddecl.ui
--- kmymoney2-0.8.orig/kmymoney2/dialogs/knewaccountwizarddecl.ui	2005-11-02 00:50:18.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/dialogs/knewaccountwizarddecl.ui	2005-11-02 00:50:44.000000000 +0000
@@ -1519,13 +1519,4 @@
     <tabstop>m_method</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kmymoneycurrencyselector.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kmymoneypayee.h</includehint>
-    <includehint>kmymoneycombo.h</includehint>
-</includehints>
 </UI>
diff -urN kmymoney2-0.8.orig/kmymoney2/dialogs/knewequityentrydecl.ui kmymoney2-0.8/kmymoney2/dialogs/knewequityentrydecl.ui
--- kmymoney2-0.8.orig/kmymoney2/dialogs/knewequityentrydecl.ui	2005-11-02 00:50:18.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/dialogs/knewequityentrydecl.ui	2005-11-02 00:50:44.000000000 +0000
@@ -233,7 +233,4 @@
     <tabstop>btnCancel</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kmymoneyedit.h</includehint>
-</includehints>
 </UI>
diff -urN kmymoney2-0.8.orig/kmymoney2/dialogs/knewinvestmentwizarddecl.ui kmymoney2-0.8/kmymoney2/dialogs/knewinvestmentwizarddecl.ui
--- kmymoney2-0.8.orig/kmymoney2/dialogs/knewinvestmentwizarddecl.ui	2005-11-02 00:50:18.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/dialogs/knewinvestmentwizarddecl.ui	2005-11-02 00:50:44.000000000 +0000
@@ -503,11 +503,4 @@
     <tabstop>m_onlineSourceCombo</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kmymoneylineedit.h</includehint>
-    <includehint>kmymoneylineedit.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneycurrencyselector.h</includehint>
-    <includehint>kmymoneylineedit.h</includehint>
-</includehints>
 </UI>
diff -urN kmymoney2-0.8.orig/kmymoney2/dialogs/knewloanwizarddecl.ui kmymoney2-0.8/kmymoney2/dialogs/knewloanwizarddecl.ui
--- kmymoney2-0.8.orig/kmymoney2/dialogs/knewloanwizarddecl.ui	2005-11-02 00:50:18.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/dialogs/knewloanwizarddecl.ui	2005-11-02 00:50:44.000000000 +0000
@@ -5546,26 +5546,4 @@
     </image>
 </images>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kmymoneylineedit.h</includehint>
-    <includehint>kmymoneypayee.h</includehint>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>knuminput.h</includehint>
-    <includehint>kmymoneycombo.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>knuminput.h</includehint>
-    <includehint>kmymoneycombo.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneyaccountselector.h</includehint>
-    <includehint>kmymoneyaccountselector.h</includehint>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kmymoneyaccountselector.h</includehint>
-    <includehint>kmymoneydateinput.h</includehint>
-</includehints>
 </UI>
diff -urN kmymoney2-0.8.orig/kmymoney2/dialogs/kofxdirectconnectdlgdecl.ui kmymoney2-0.8/kmymoney2/dialogs/kofxdirectconnectdlgdecl.ui
--- kmymoney2-0.8.orig/kmymoney2/dialogs/kofxdirectconnectdlgdecl.ui	2005-11-02 00:50:18.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/dialogs/kofxdirectconnectdlgdecl.ui	2005-11-02 00:50:44.000000000 +0000
@@ -106,7 +106,4 @@
     </connection>
 </connections>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kprogress.h</includehint>
-</includehints>
 </UI>
diff -urN kmymoney2-0.8.orig/kmymoney2/dialogs/konlinequoteconfigurationdecl.ui kmymoney2-0.8/kmymoney2/dialogs/konlinequoteconfigurationdecl.ui
--- kmymoney2-0.8.orig/kmymoney2/dialogs/konlinequoteconfigurationdecl.ui	2005-11-02 00:50:18.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/dialogs/konlinequoteconfigurationdecl.ui	2005-11-02 00:50:44.000000000 +0000
@@ -196,10 +196,4 @@
     </connection>
 </connections>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kcombobox.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kmymoney2-0.8.orig/kmymoney2/dialogs/ksplitcorrectiondlg.ui kmymoney2-0.8/kmymoney2/dialogs/ksplitcorrectiondlg.ui
--- kmymoney2-0.8.orig/kmymoney2/dialogs/ksplitcorrectiondlg.ui	2005-11-02 00:50:18.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/dialogs/ksplitcorrectiondlg.ui	2005-11-02 00:50:44.000000000 +0000
@@ -229,8 +229,4 @@
 <customwidgets>
 </customwidgets>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kmymoney2-0.8.orig/kmymoney2/dialogs/ksplittransactiondlgdecl.ui kmymoney2-0.8/kmymoney2/dialogs/ksplittransactiondlgdecl.ui
--- kmymoney2-0.8.orig/kmymoney2/dialogs/ksplittransactiondlgdecl.ui	2005-11-02 00:50:18.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/dialogs/ksplittransactiondlgdecl.ui	2005-11-02 00:50:44.000000000 +0000
@@ -382,7 +382,4 @@
     </image>
 </images>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kmymoneysplittable.h</includehint>
-</includehints>
 </UI>
diff -urN kmymoney2-0.8.orig/kmymoney2/dialogs/kupdatestockpricedlgdecl.ui kmymoney2-0.8/kmymoney2/dialogs/kupdatestockpricedlgdecl.ui
--- kmymoney2-0.8.orig/kmymoney2/dialogs/kupdatestockpricedlgdecl.ui	2005-11-02 00:50:18.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/dialogs/kupdatestockpricedlgdecl.ui	2005-11-02 00:50:44.000000000 +0000
@@ -310,9 +310,4 @@
     </image>
 </images>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kmymoneycurrencyselector.h</includehint>
-    <includehint>kmymoneysecurityselector.h</includehint>
-</includehints>
 </UI>
diff -urN kmymoney2-0.8.orig/kmymoney2/dialogs/mymoneyqifprofileeditordecl.ui kmymoney2-0.8/kmymoney2/dialogs/mymoneyqifprofileeditordecl.ui
--- kmymoney2-0.8.orig/kmymoney2/dialogs/mymoneyqifprofileeditordecl.ui	2005-11-02 00:50:18.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/dialogs/mymoneyqifprofileeditordecl.ui	2005-11-02 00:50:44.000000000 +0000
@@ -871,21 +871,4 @@
     <tabstop>m_editOutputFilterLocation</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>klineedit.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kcombobox.h</includehint>
-    <includehint>kcombobox.h</includehint>
-    <includehint>kcombobox.h</includehint>
-    <includehint>kcombobox.h</includehint>
-    <includehint>kurlrequester.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kurlrequester.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kmymoney2-0.8.orig/kmymoney2/views/kbankviewdecl.ui kmymoney2-0.8/kmymoney2/views/kbankviewdecl.ui
--- kmymoney2-0.8.orig/kmymoney2/views/kbankviewdecl.ui	2005-11-02 00:50:18.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/views/kbankviewdecl.ui	2005-11-02 00:50:44.000000000 +0000
@@ -196,8 +196,4 @@
     </image>
 </images>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kmymoneytitlelabel.h</includehint>
-    <includehint>kmymoneyaccounttree.h</includehint>
-</includehints>
 </UI>
diff -urN kmymoney2-0.8.orig/kmymoney2/views/kcategoriesviewdecl.ui kmymoney2-0.8/kmymoney2/views/kcategoriesviewdecl.ui
--- kmymoney2-0.8.orig/kmymoney2/views/kcategoriesviewdecl.ui	2005-11-02 00:50:18.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/views/kcategoriesviewdecl.ui	2005-11-02 00:50:44.000000000 +0000
@@ -165,8 +165,4 @@
 <tabstops>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kmymoneyaccounttree.h</includehint>
-    <includehint>kmymoneytitlelabel.h</includehint>
-</includehints>
 </UI>
diff -urN kmymoney2-0.8.orig/kmymoney2/views/kinvestmentviewdecl.ui kmymoney2-0.8/kmymoney2/views/kinvestmentviewdecl.ui
--- kmymoney2-0.8.orig/kmymoney2/views/kinvestmentviewdecl.ui	2005-11-02 00:50:18.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/views/kinvestmentviewdecl.ui	2005-11-02 00:50:44.000000000 +0000
@@ -173,8 +173,4 @@
     </image>
 </images>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kmymoneyaccountcombo.h</includehint>
-    <includehint>kmymoneytitlelabel.h</includehint>
-</includehints>
 </UI>
diff -urN kmymoney2-0.8.orig/kmymoney2/views/kpayeesviewdecl.ui kmymoney2-0.8/kmymoney2/views/kpayeesviewdecl.ui
--- kmymoney2-0.8.orig/kmymoney2/views/kpayeesviewdecl.ui	2005-11-02 00:50:18.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/views/kpayeesviewdecl.ui	2005-11-02 00:50:44.000000000 +0000
@@ -535,7 +535,4 @@
     </image>
 </images>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kmymoneytitlelabel.h</includehint>
-</includehints>
 </UI>
diff -urN kmymoney2-0.8.orig/kmymoney2/views/kscheduledviewdecl.ui kmymoney2-0.8/kmymoney2/views/kscheduledviewdecl.ui
--- kmymoney2-0.8.orig/kmymoney2/views/kscheduledviewdecl.ui	2005-11-02 00:50:18.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/views/kscheduledviewdecl.ui	2005-11-02 00:50:44.000000000 +0000
@@ -190,12 +190,4 @@
     </image>
 </images>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>qwidget.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>klistview.h</includehint>
-    <includehint>kmymoneyscheduledcalendar.h</includehint>
-    <includehint>kmymoneytitlelabel.h</includehint>
-</includehints>
 </UI>
diff -urN kmymoney2-0.8.orig/kmymoney2/views/kstocktransactionviewdecl.ui kmymoney2-0.8/kmymoney2/views/kstocktransactionviewdecl.ui
--- kmymoney2-0.8.orig/kmymoney2/views/kstocktransactionviewdecl.ui	2005-11-02 00:50:18.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/views/kstocktransactionviewdecl.ui	2005-11-02 00:50:45.000000000 +0000
@@ -62,7 +62,4 @@
     </vbox>
 </widget>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>qwidget.h</includehint>
-</includehints>
 </UI>
diff -urN kmymoney2-0.8.orig/kmymoney2/widgets/kmymoneyonlinequoteconfigdecl.ui kmymoney2-0.8/kmymoney2/widgets/kmymoneyonlinequoteconfigdecl.ui
--- kmymoney2-0.8.orig/kmymoney2/widgets/kmymoneyonlinequoteconfigdecl.ui	2005-11-02 00:50:19.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/widgets/kmymoneyonlinequoteconfigdecl.ui	2005-11-02 00:50:45.000000000 +0000
@@ -228,12 +228,4 @@
 <customwidgets>
 </customwidgets>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>klistview.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>klineedit.h</includehint>
-</includehints>
 </UI>
diff -urN kmymoney2-0.8.orig/kmymoney2/widgets/kmymoneypriceviewdecl.ui kmymoney2-0.8/kmymoney2/widgets/kmymoneypriceviewdecl.ui
--- kmymoney2-0.8.orig/kmymoney2/widgets/kmymoneypriceviewdecl.ui	2005-11-02 00:50:19.000000000 +0000
+++ kmymoney2-0.8/kmymoney2/widgets/kmymoneypriceviewdecl.ui	2005-11-02 00:50:45.000000000 +0000
@@ -32,7 +32,4 @@
 <customwidgets>
 </customwidgets>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>klistview.h</includehint>
-</includehints>
 </UI>
