--- kmymoney-5.1.2/kmymoney/plugins/kbanking/kbanking.h~	2021-06-23 17:49:12.000000000 +0200
+++ kmymoney-5.1.2/kmymoney/plugins/kbanking/kbanking.h	2021-10-27 00:48:05.130776795 +0200
@@ -158,7 +158,7 @@
       *
       * @deprecated
       */
-    bool updateAccount(const MyMoneyAccount& acc) DEPRECATED;
+    bool updateAccount(const MyMoneyAccount& acc) QT_DEPRECATED;
 
     /**
       * Trigger the password cache timer
